### PR TITLE
Add myself as maintainer

### PR DESCRIPTION
**Pull request by jan-janssen**
Checklist
* [ ] Used a fork of the feedstock to propose changes
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy`
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by jan-janssen** [comment]
@conda-forge-admin, please rerender

**Comment by jan-janssen** [comment]
Waiting for https://github.com/conda-forge/oauthlib-feedstock/pull/15 to be merged first.

**Comment by jan-janssen** [comment]
@conda-forge-admin restart ci

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,45 +1,46 @@
...
+  sha256: 5bb79c8a8e670c2eb4cb553dfc3283b6c8d1202f674934676dc173cee94fe39c
 
 build:
   noarch: python
   number: 0
-  script: python -m pip install --no-deps --ignore-installed .
+  script: {{ PYTHON }} -m pip install . -vv
 
 requirements:
   build:
```
Comment (beckermr) [inline recipe/meta.yaml]: This should be `host`.

Comment (beckermr) [inline recipe/meta.yaml]:
```suggestion
  host:
```

**Comment by beckermr** [comment]
@conda-forge-admin rerender

**Pull request merged**